nsqd/http.go

```diff
@@ -596,3 +600,41 @@ func (s *httpServer) printStats(stats []TopicStats, health string, startTime tim
 	}
 	return buf.Bytes()
 }
+
+type allowedOpts struct {
+	NSQLookupdTCPAddresses []string `json:"nsqlookupd_tcp_addresses"`
```

Comment:
I understand why that would be valuable but the API feels wrong.  The requirements really argue for _specific_ `nsqlookupd` config endpoints, which I was hoping to avoid.